package.json

```diff
@@ -24,6 +24,7 @@
     "@shopify/remix-oxygen": "^1.0.3",
     "@types/styled-components": "^5.1.26",
     "clsx": "^1.2.1",
+    "cross-env": "^7.0.3",
```

Comment:
I'm not sure how the project requires this.

Reply:
I used "cross-env" because I use Windows OS and I had some issues running Storybook and I found this module searching the solution